app/routes/missions._index.mdx

```diff
@@ -23,6 +23,7 @@ Observatories that generate GCN notices for transient alerts, as consumed by the
 | [Fermi](missions/fermi)                                          | Operational | [Archive](https://gcn.gsfc.nasa.gov/fermi_grbs.html), <br/> <SelectedAlertQuickstartLink alertKey={"Fermi"} format="text" />,<br/><SelectedAlertEmailLink alertKey={"Fermi"} format="text" />                                                                                     | JSON in development                                                                                             |
 | [GECAM](missions/gecam)                                          | Operational | [Archive](https://gcn.gsfc.nasa.gov/gecam_events.html), <br/> <SelectedAlertQuickstartLink alertKey={"GECAM"} format="text" />, <br/><SelectedAlertEmailLink alertKey={"GECAM"} format="text" />                                                                                  |                                                                                                                 |
 | [HAWC](missions/hawc)                                            | Operational | [Archive](https://gcn.gsfc.nasa.gov/amon_hawc_events.html), <br/> <SelectedAlertQuickstartLink alertKey={"Other"} format="text" otherAlerts={[ 'HAWC_BURST_MONITOR' ]} />,<br/><SelectedAlertEmailLink alertKey={"Other"} format="text" otherAlerts={[ 'HAWC_BURST_MONITOR' ]} /> |                                                                                                                 |
+| HEASARC                                                          | Onboarding  | [Archive](https://heasarc.gsfc.nasa.gov)                                                                                                                                                                                                                                          |                                                                                                                 |
```

Comment:
The other archive links are archives of the Notices.  While HEASARC itself is an archive, we don't yet have an archive of notices that haven't yet been produced.  So we should just remove that archive link.